PythonPonies/bangazon-storefront
# Products Model, and Passing Test

# Description
This pull request adds tests for the products model, product details view, and navigation view. All tests started as failing and now the Product Model Test passes.

## Number of Fixes
No fixes, just new code.

## Related Ticket(s)
#8 #9 

## Problem to Solve
We need a Products model to build out the rest of the app.

## Proposed Changes
I added a Products model.

## Expected Behavior
3 tests should fail, but the `test_products_can_be_added_to_database` test will pass.

## Steps to Test Solution

1. Fetch this Branch
2. Run `python manage.py test bangazon_storefront`
3. The `test_products_can_be_added_to_database` test will pass.

3 tests should fail. I still need to write code for the`test_navigation_view` test and `test_product_details_view` test, and Ike is still writing code to pass the `test_get_all_product_types` test.

## Testing

[x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[x] I added documentation for any new classes/methods

## Deploy Notes
none
